app/metrics.yaml

```diff
@@ -3131,3 +3131,61 @@ perf.awesomebar:
       - fenix-core@mozilla.com
       - gkruglov@mozilla.com
     expires: "2020-10-01"
+
+progressive_web_app:
+  homescreen_tap:
+    type: event
+    description: |
+      A user taps on PWA homescreen icon
+    bugs:
+      - https://github.com/mozilla-mobile/fenix/issues/10261
+    data_reviews:
+      - https://github.com/mozilla-mobile/fenix/pull/11859
+    notification_emails:
+      - fenix-core@mozilla.com
+      - erichards@mozilla.com
+    expires: "2020-10-01"
+  install_tap:
+    type: event
+    description: |
+      A user installs a PWA. Could be a shortcut or added to homescreen.
+    bugs:
+      - https://github.com/mozilla-mobile/fenix/issues/10261
+    data_reviews:
+      - https://github.com/mozilla-mobile/fenix/pull/11859
+    notification_emails:
+      - fenix-core@mozilla.com
+      - erichards@mozilla.com
+    expires: "2020-10-01"
+  foreground:
+    type: event
+    description: |
+      An intent receiver activity is foregrounded for PWA.
+    extra_keys:
+      duration:
+        description: |
+          The current time in ms when the intent was foregrounded.
```

Comment:
Is the idea that DS will do this calculation themselves? Can you verify with Marissa that that is the case, or if the client should calculate that duration between foreground/background when backgrounding the activity?

Reply:
This is all done in AC and they don't want calculations like this to happen 😢

Reply:
Also we haven't defined what time we want between foreground/bg, so Marissa can change it as needed without needing a code fix